content/news/playtest-20151129.md

```diff
@@ -0,0 +1,33 @@
+We received a lot of feedback on our last two playtests, and we tried to address as many of the reported issues as possible.  Thanks to everyone who has tried the playtests and reported bugs or other feedback!
+
+We fixed some new, as well as some long-standing issues for this playtest, so hopefully this will be the final release candidate for the upcoming December release.  We encourage you to give this release a good try out, especially if you like to play competitively online, because if we don't hear your balance complaints now then you will be stuck with them for the next few months until our next release!  You can report any issues you have by [filing a bug](http://bugs.openra.net) or [joining our IRC channel](/community/).
+
+The focus for this playtest was on:
+
+* Fixing bugs with target acquisition with GPS and Gap Generators.
+* Fixing issues with some Tiberian Dawn and Red Alert missions.
+* Polishing the Dune 2000 thumpers, spiceblooms, Saboteurs and production structure decorations.
+* Fixing some crashes.
+
+In particular, we responded to the following issues reported during the last playtest:
+
+* Fixed a crash in the GDI06 mission. (thanks, @Snurre86)
+* Fixed a crash when refreshing the multiplayer server list. (thanks, @pi99y)
+* Fixed a crash when spawning resources outside the map borders. (thanks, @Frount)
+* Fixed an unintentional buff to missiles across all mods. (thanks, @Spy)
+* Restored targeting cursor when hovering over disguised enemy spies. (thanks, @kyrylo)
+
+As usual, you can find the full details about the changes on the [changelog page](https://github.com/OpenRA/OpenRA/wiki/Changelog) and the installer for your operating system on our [download page](/download/)!
```

Comment:
Permalink to the changelog: https://github.com/OpenRA/OpenRA/wiki/Changelog/36203a1fdc608d380964485bac9d8671a5658983